app/tests/spec/views/complete_reset_password.js

```diff
@@ -361,9 +364,12 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
                   EMAIL,
                   PASSWORD,
                   relier,
+                  // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+                  /* eslint-disable */
```

Comment:
Yeah... I also don't see how it's fixed by the linked issue.

Can it be fixed now with more/less indent?